docker-compose/sdp-docker-compose.yml

```diff
@@ -26,22 +26,41 @@ services:
     entrypoint: ["python", "SDPMaster"]
     command: ["1", "-v4"]
 
-  sdpsubarray:
+
+  sdpsubarray1:
     image: ${DOCKER_REGISTRY_HOST}/sdp-prototype/tangods_sdp_subarray:0.5.16
     depends_on:
       - databaseds
       - rsyslog-tmcprototype
       - sdp-dsconfig
-    container_name: ${CONTAINER_NAME_PREFIX}sdp_subarray
+    container_name: ${CONTAINER_NAME_PREFIX}sdp_subarray1
```

Comment:
There is a need to come up with a standard for container naming, similar to device naming standards.